content/software-security/cves/cve-remediation.md

```diff
@@ -0,0 +1,85 @@
+
+* **False positives** – Though vulnerability scanners may find and report a CVE for a given dependency, it may not be present in your software. If the vulnerability is outside of your program’s scope (for example, in a function that you are not calling) then the vulnerability may not truly impact your application, and you do not need to spend time addressing it.
+
+* **Vulnerability severity** – CVEs are assigned a CVSS score to assess the severity of their impacts if exploited. A CVE with a CVSS score of critical may have serious consequences to the [confidentiality, integrity, and availability](https://www.cisecurity.org/insights/spotlight/ei-isac-cybersecurity-spotlight-cia-triad) of a system, making it a priority to remediate. In comparison, a CVE with a low severity score may be difficult to exploit or has minimal impact, so it can be remediated after higher priorities are addressed.
+
+* **Exploitability status** – Not all vulnerabilities have been observed to be exploited. Checking to see if a vulnerability is in the [Known Exploited Vulnerabilities (KEV) Catalog](https://www.cisa.gov/known-exploited-vulnerabilities-catalog) can tell you if there is an active attacker threat “in the wild” for vulnerabilities present in your software. Note that a vulnerability’s absence from the KEV Catalog does not mean it won’t be exploitable, as unreported or unobserved attacks on a vulnerability may still exist (or may occur in the future).
+
+[Vulnerability Exploitability eXchange (VEX)](https://www.cisa.gov/sites/default/files/2023-01/VEX_Use_Cases_Aprill2022.pdf) documents can be helpful references when triaging vulnerabilities. VEX is a model which allows software developers to report the status of vulnerabilities in a software product to inform downstream users of what actions they should take to address them. If a scanner reports a vulnerability in a product, a VEX document can elaborate on how the vulnerability truly impacts it.
+
+
+## Step 4 - Remediation
+
+Once you have triaged the vulnerabilities found in your software, you can move on to remediating them. Vulnerability *remediation* is the act of correcting and removing CVEs from your software. Using a CVE ID, you can check sources such as the [National Vulnerability Database (NVD)](https://nvd.nist.gov/) for advisories regarding a vulnerability. Here, you can learn what steps should be taken to address the vulnerability. Oftentimes, this will include updating dependencies to patched versions. After you have followed guidance to fix the vulnerability, you should scan your software again to ensure that it was properly remediated.
+
+
+## Vulnerabilities in Container Images
```

Comment:
If someone could pass over the container images section, that would be great. I made a few changes based on feedback, and tried to reference Alpine, though I'm not sure how effective it is.